src/main/java/com/teamacronymcoders/base/subblocksystem/blocks/BlockSubBlockHolder.java

```diff
@@ -164,4 +167,75 @@ public int damageDropped(@Nonnull IBlockState state) {
     public List<IGeneratedModel> getGeneratedModels() {
         return this.getSubBlocks().values().stream().map(ISubBlock::getGeneratedModel).filter(Objects::nonNull).collect(Collectors.toList());
     }
+
+    @Override
+    public boolean isSideSolid(IBlockState blockState, @Nonnull IBlockAccess world, @Nonnull BlockPos pos, EnumFacing side) {
+        return this.getSubBlock(blockState).isSideSolid();
+    }
+
+    @Override
+    public boolean isTopSolid(IBlockState blockState) {
```

Comment:
Does this not delegate to isSideSolid? (In the Block class?)

Reply:
```Java
public boolean isTopSolid(IBlockState state)
{
    return state.getMaterial().isOpaque() && state.isFullCube();
}
```